lms/djangoapps/gating/tests/test_signals.py

```diff
@@ -19,32 +19,26 @@ def setUp(self):
         super(TestHandleScoreChanged, self).setUp()
         self.course = CourseFactory.create(org='TestX', number='TS01', run='2016_Q1')
         self.user = UserFactory.create()
-        self.test_usage_key = self.course.location
+        self.subsection_grade = Mock()
 
-    @patch('gating.signals.gating_api.evaluate_prerequisite')
-    def test_gating_enabled(self, mock_evaluate):
-        """ Test evaluate_prerequisite is called when course.enable_subsection_gating is True """
+    @patch('lms.djangoapps.gating.api.gating_api.get_gating_milestone')
+    def test_gating_enabled(self, mock_gating_milestone):
         self.course.enable_subsection_gating = True
         modulestore().update_item(self.course, 0)
-        handle_score_changed(
+        evaluate_subsection_gated_milestones(
             sender=None,
-            points_possible=1,
-            points_earned=1,
-            user_id=self.user.id,
-            course_id=unicode(self.course.id),
-            usage_id=unicode(self.test_usage_key)
+            user=self.user,
+            course=self.course,
+            subsection_grade=self.subsection_grade,
         )
-        mock_evaluate.assert_called_with(self.course, self.course, self.user.id)  # pylint: disable=no-member
```

Comment:
**Note to reviewers:**  This test never really worked since `assert_called_with` was essentially a no-op mock method on a mock object - and never really asserted.